middleware/src/middleware.go

```diff
@@ -251,6 +251,26 @@ func (middleware *Middleware) Restore(method rpcmessages.RestoreArgs) (rpcmessag
 	}
 }
 
+// Authenticate returns an GenericResponse struct in response to a rpcserver request
+// FIXME: This is a dummy implementation of the Authenticate RPC call
+// in the future this should return an AuthentificationResponse with e.g. an JWT
+func (middleware *Middleware) Authenticate(args rpcmessages.AuthenticateArgs) (rpcmessages.GenericResponse, error) {
+	dummyUsers := map[string]string{
+		"satoshi": "shift1",
+		"sipa":    "shift6",
+		"djb":     "shift9",
+	}
+
+	if expectedPasssword, userIsInMap := dummyUsers[args.Username]; userIsInMap {
```

Comment:
Personal preference to use a custom variable instead of just `ok` which is generally the case in Go?
I don't really mind either way, just wondering :)

Reply:
Had `ok` at first, but wanted the code to be clear(er), when dealing with passwords. But your right, I'd normally expect a `ok` as well.